rules/false_positives/wolfictl.yara

```diff
@@ -0,0 +1,14 @@
+rule wolfictl_binary : override {
+  meta:
+    curl_tor_chmod_relative_run = "high"
```

Comment:
medium

Reply:
I'm starting to see a pattern here: I think `curl_tor_chmod_relative_run` should be made "high" for now. It's too noisy for critical.

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).